js/exercise-02.js

```diff
@@ -0,0 +1,18 @@
+/* 
+    1. Declare a variable using let with any name you like, but without a value.
+    2. On a separate line, assign the variable the String: "Moo!"
+    3. Log out this variable to the terminal using console.log
+    
+    Repeat the above steps but try using a const variable. Comment this new part of code and explain what is happening.
+*/
+
+//El que esta passant en el troç de codi següent és:
+//Declarem ambdues variables tot i aixi nomes la variable 'duckLet' passa per la 'Death Zone' p
+
+let duckLet; //Declarem la varialbe 'duckLet' i automàticament passa a ser a la 'Death Zone'
+duckLet = "Moo"; // Inicialitzem la variable, surt de la 'Death Zone' 
+console.log(duckLet); //Imprimim el valor de la variable per consola
+
+const duckConst= "Moo"; //Declarem la varialble, pero aquest tipus de variable ens obliga a inicialitzarla al moment o en cas contrari la consola peta.
```

Comment:
It is JS itself the one is causing an error on runtime.

The console does not crash... it is only logging the error.